sites/public/src/pages/account/StatusItemWrapper.tsx

```diff
@@ -11,11 +11,11 @@ interface StatusItemWrapperProps {
 }
 
 const StatusItemWrapper = (props: StatusItemWrapperProps) => {
+  const applicationDueDate = props.application?.fullListing?.applicationDueDate
+
   return (
     <StatusItem
-      applicationDueDate={dayjs(props.application?.fullListing?.applicationDueDate).format(
-        "MMMM D, YYYY"
-      )}
+      applicationDueDate={!applicationDueDate && dayjs(applicationDueDate).format("MMMM D, YYYY")}
```

Comment:
This is sometimes returning Invalid Date
![image](https://github.com/bloom-housing/bloom/assets/42942267/9d7f5d80-8670-49c1-b5c4-47cd133adb54)

Reply:
Ah good catch, I had that "!" in there for testing. Should work now 🐬